engine/source/runtime/function/render/include/render/surface.h

```diff
@@ -45,6 +45,9 @@ namespace Pilot
         void draw_frame();
         int  clear();
         void setDefaultStyle();
+    protected:
+        float contentScale();
```

Comment:
suggest naming: `float getConentScale() const` and `float getIndentScale() const`